### PR TITLE
Feature/add additional opts

### LINKED ISSUE #10: Pass arguments to lftp
Feature request:

Whilst using your package, I ran into an issue. I publish files to my ftp server, into a specific folder. However, the files that are already there are either overridden (if their name is the same as a new file) or they stay as they are, which hogs my storage.

That is why I have to delete all files from the target directory prior to transferring. Incidentally, lftp has a flag for this exact situation, namely the —delete flag.

Could the user be granted a property with which they could pass arguments to lftp, such as —delete?

Another solution is to make a boolean flag for the —delete flag specifically, which controls whether or not the —delete flag is added as an argument to lftp.

### LINKED ISSUE #15: Options to remove files from remote dir
Options to delete the entire directory before upload ou to sync files, removing files that doesn't existis anymore.

### PR DESCRIPTION
**Pull request by hsimah**
Allow users to pass in any additional mirror command options they need. I need to run `--delete` in addition

### CODE REVIEW

**Comment by hsimah** [comment]
I figured this would be more extensible than mapping all lftp options to inputs.

File: README.md [hunk 1]
```diff
@@ -14,13 +14,14 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Upload ftp
-      uses: sebastianpopp/ftp-action@releases/v2
+      uses: hsimah/ftp-action-delete-first@releases/v2
```
Comment (sebastianpopp) [inline README.md]: Could you revert this?

File: action.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-name: 'ftp-action'
-author: 'Sebastian Popp <ftp-action@sebastianpopp.com>'
+name: 'ftp-action-delete-first'
+author: 'hsimah'
```
Comment (sebastianpopp) [inline action.yml]: ... and revert `name` and `author`

**Review by sebastianpopp — COMMENTED** [review]
Thank you for your contribution! I'd be happy to merge if you could just revert back the name and author (see my comments).

File: action.yml (hunk 2 above)
Comment (hsimah) [reply action.yml]: 🤦 I did not mean to include these in the PR; let me revert that commit

**Comment by hsimah** [comment]
sorry for being careless with my commits!

**Comment by sebastianpopp** [comment]
Thanks!

**Pull request merged**